layers/+lang/ipython-notebook/packages.el

```diff
@@ -68,7 +68,9 @@
         "+" 'ein:notebook-worksheet-insert-next
         "-" 'ein:notebook-worksheet-delete
         "x" 'ein:notebook-close
-        "u" 'ein:worksheet-change-cell-type
+        "u" 'ein:worksheet-change-cell-type)
+
+      (evil-leader/set-key-for-mode 'ein:notebook-multilang-mode
```

Comment:
Ups, I did this in a hurry. It seems that there no longer is a function that can do what we need here. I don't have time right now to dive into the spacemacs keybinding logic to set this up so it works with all the editing modes.